charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.13.3
 kubeVersion: ">=1.25.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 7.7.18
+version: 7.7.20
```

Comment:
...🤔

Reply:
The script always used to always increment by 1 every time it’s invoked. I suspect that’s all that’s happened here.  The rule always used to be “only let the renovate script run once per pr”

Reply:
You can manually set the version back down and it should be fine.

Reply:
I see 👀 
I fixed the version. 🙋 https://github.com/argoproj/argo-helm/pull/3108/commits/a77021925efd662c93aa1fafe0bcbfa1ca8c7b7c

Reply:
Ah damn. I'll try to handle this in a separate PR / issue. It is introduced by my recent Renovate improvement PR:
- https://github.com/argoproj/argo-helm/pull/3143

Reply:
I think it has always done it. It’s just been run more than once on this PR. 

It increments whatever number is on the branch rather than looking at the base branch’s value.